rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -81,150 +83,293 @@ typedef struct
     RotationBatch rotation_batch;
 }JointsDataBatch;
 
+typedef struct MetaDataInfo {
+public:
+    uint _img_id = -1;
+    std::string _img_name = "";
+    ImgSize _img_size = {};
```

Comment:
```suggestion
    uint _img_id = -1;
    std::string _img_name = "";
    ImgSize _img_size = {};
```
If these are public variables, I think we need not use _ in the beginning of variable name